lib/orcid/client.rb

```diff
@@ -41,8 +43,11 @@ def add_work(orcidid, work, token)
 
     private
 
-    def base_orcidid(orcidid)
-      orcidid[-19, 19]
+    def get(url)
+      response = conn.get(url)
+      raise "ORCID.org API returned #{response.status}" if response.status != 200
```

Comment:
just double-checking ... will this give us an HB alert?

Reply:
If not caught and used in an environment in which HB is configured, I'd expect so.